packages/cdktf/test/helper/resource.ts

```diff
@@ -18,6 +23,7 @@ export class TestResource extends TerraformResource {
   public names?: string[];
   public tags?: { [key: string]: string };
   public nestedType?: { [key: string]: string };
+  public listBlock?: IResolvable; // TODO: add special type?
```

Comment:
TODO left

Reply:
changed to a better wording, same as in the data source